yt_dlp/output/logging_output.py

```diff
@@ -0,0 +1,137 @@
+import inspect
+import logging
+import sys
+import traceback
+
+
+class deferred_class_attribute:
+    def __init__(self, func):
+        self.func = func
+
+    def __get__(self, _, owner):
+        result = self.func()
+        setattr(owner, self.func.__name__, result)
+        return result
+
+
+class _YDLIgnoringLogger(logging.Logger):
+    @deferred_class_attribute
+    def _internal_frames():
+        from . import logging as output_logging
+        from ..downloader import common as downloader
+        from ..extractor import common as extractor
+
+        return frozenset(map(inspect.getabsfile, [
+            _YDLIgnoringLogger,
+            output_logging,
+            logging,
+            downloader,
+            extractor,
+        ]))
```

Comment:
This is a set of files which count as "internal" (they call logging functions but might not be the root cause). Are these all the cases?